src/app/pages/TopicPage/Pagination/index.jsx

```diff
@@ -1,64 +1,81 @@
 import React, { useContext } from 'react';
 import styled from '@emotion/styled';
 import { number } from 'prop-types';
-import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { getSansRegular, getSansBold } from '@bbc/psammead-styles/font-styles';
 import {
-  C_DIM_GREY,
-  C_EBON,
-  C_GREY_11,
   C_PHILIPPINE_GREY,
-  C_WHITE,
+  C_GREY_10,
+  C_POSTBOX,
 } from '@bbc/psammead-styles/colours';
 
 import {
   GEL_GROUP_2_SCREEN_WIDTH_MIN,
-  GEL_GROUP_2_SCREEN_WIDTH_MAX,
   GEL_GROUP_3_SCREEN_WIDTH_MIN,
-  GEL_GROUP_3_SCREEN_WIDTH_MAX,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
 } from '@bbc/gel-foundations/breakpoints';
 import { ServiceContext } from '#contexts/ServiceContext';
 
-import buildBlocks, { TYPE, VISIBILITY, AVAILABILITY } from './buildBlocks';
+import buildBlocks, { TYPE, VISIBILITY } from './buildBlocks';
 import { Ellipsis, LeftChevron, RightChevron } from './icons';
 
 const visibilityToMediaQuery = visibility =>
   ({
-    // TODO: minimum width being specified here as a future PR is implementing a totally different design for group 1
-    [VISIBILITY.MOBILE_ONLY]: `@media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    [VISIBILITY.MOBILE_ONLY]: `display: none; @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
       display: inline-block;
     }`,
-    // TODO: minimum width being specified here as a future PR is implementing a totally different design for group 1
-    [VISIBILITY.TABLET_DOWN]: `@media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    [VISIBILITY.TABLET_DOWN]: `display: none; @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
       display: inline-block;
     }`,
-    [VISIBILITY.TABLET_UP]: `@media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    [VISIBILITY.TABLET_UP]: `display: none; @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
       display: inline-block;
     }`,
-    [VISIBILITY.DESKTOP_ONLY]: `@media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    [VISIBILITY.DESKTOP_ONLY]: `display: none; @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
       display: inline-block;
     }`,
-  }[visibility] || '');
+    [VISIBILITY.ALL]: `display: inline-block;`,
+  }[visibility] || 'display: none;');
 
+const Nav = styled.nav`
+  display: block;
+  margin: 0 auto 2.5rem auto;
+  text-align: center;
+`;
 const StyledOrderedList = styled.ol`
```

Comment:
I think this should be an unordered list as defined in the screen reader UX?

Reply:
👍  Updated